test/quality/Makefile

```diff
@@ -34,15 +33,9 @@ validate: venv $(VULNERABILITY_LABELS)/Makefile ## Run all quality checks agains
 capture: sboms vulns ## Collect and store all syft and grype results
 
 .PHONY: capture
-vulns: venv $(TEST_DB) check-db ## Collect and store all grype results
+vulns: venv $(TEST_DB) ## Collect and store all grype results
 	$(YARDSTICK) -v result capture -r $(RESULT_SET)
 
-.PHONY: check-db
-check-db:
```

Comment:
Do we need to set a DB max age in yardstick config or something to ensure that we don't run grype quality against a super old DB forever?